nimbus.fml.yaml

```diff
@@ -20,6 +20,10 @@ features:
         description: If `true`, the app will show the cfrs
         type: Boolean
         default: false
+      is-promote-search-widget-dialog-enabled:
```

Comment:
suggestion: Could help enabling these new features in debug. Can be done with adding at the same indentation levels with `variables`:
```yaml
  defaults:
    - channel: debug
      value: {
        "variables": {
          "is-enabled": true,
          "is-cfr-enabled": true,
          "is-promote-search-widget-dialog-enabled": true,
        }
      }
```

Reply:
Done